### ISSUE TITLE
ArrayIndexOutOfBoundsException: 258

### ISSUE THREAD

**Issue by tpenguinltg** [itdelatrisu/opsu#261]
I was playing Lanturn's [Endless Tewi-ma Park](https://osu.ppy.sh/s/90935) 2B difficulty and the game crashed at the end of the map. I might have trouble reproducing this again because I only got it when I was using a tablet (thus performing better), but I no longer have access to said tablet.

On commit e3a4d37.

* * *
**Version:** 0.15.0
**Build date:** 2017-02-09 21:35
**OS:** Linux (amd64)
**JRE:** 1.8.0_121
**OpenGL version:** 3.0 Mesa 13.0.3 (Intel Open Source Technology Center)
**Stack trace:**
```
java.lang.ArrayIndexOutOfBoundsException: 258
	at itdelatrisu.opsu.states.Game.sendGameKeyPress(Game.java:2115)
	at itdelatrisu.opsu.states.Game.runReplayFrame(Game.java:2083)
	at itdelatrisu.opsu.states.Game.addReplayFrameAndRun(Game.java:2070)
	at itdelatrisu.opsu.states.Game.gameKeyPressed(Game.java:1339)
	at itdelatrisu.opsu.states.Game.mousePressed(Game.java:1299)
	at org.newdawn.slick.state.StateBasedGame.mousePressed(StateBasedGame.java:499)
	at org.newdawn.slick.Input.poll(Input.java:1241)
	at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:680)
	at itdelatrisu.opsu.Container.gameLoop(Container.java:135)
	at itdelatrisu.opsu.Container.start(Container.java:79)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:216)
```

**Comment by tpenguinltg** [comment]
Game.java:2115 is line 2117 in the latest commit:
```
// check current object
HitObject hitObject = beatmap.objects[objectIndex];
```

**Comment by itdelatrisu** [comment]
Fixed in 1885016, thank you!
